doc/filters/data_uri.rst

```diff
@@ -25,15 +25,15 @@ The ``data_uri`` filter generates a URL using the data scheme as defined in RFC
 
 .. note::
 
-    The ``data_uri`` filter is part of the ``HtmlExtension`` which is not
-    installed by default. Install it first:
+    The ``data_uri`` filter is part of the [``HtmlExtension``](https://github.com/twigphp/html-extra)
+    which is not installed by default. Install it first:
```

Comment:
I would not add links to the extra bundle Github repository as instructions are just below.

Reply:
Well, below are instructions on how to *install* it, but no explanation what this thing actually *is* ;-)

So do you still want me to remove it? It's certainly not worth an elaborate discussion... :-)

Reply:
Yes, I know, I prefer to remove it.